internal/blockchain/chain.go

```diff
@@ -1592,11 +1600,11 @@ func (b *BlockChain) BestSnapshot() *BestState {
 //
 // This function MUST be called with the chain state lock held (for reads).
 func (b *BlockChain) maxBlockSize(prevNode *blockNode) (int64, error) {
-	// Determine the correct deployment version for the block size consensus
+	// Determine the correct deployment details for the block size consensus
```

Comment:
I think this comment might be wrong. Should it be `treat it as **not active** when voting is not enabled`?

When `ok == false` the value returned is `MaximumBlockSizes[0]`. Looking at the code below, that seems to be the value returned when the agenda is not active.

Reply:
Correct.  The comment was already incorrect.  Unlike most other cases, the max block size one does treat it as **not active** when voting isn't enabled.

Updated.